src/doc/src/reference/workspaces.md

```diff
@@ -57,6 +65,10 @@ version = "0.1.0"    # the current version, obeying semver
 authors = ["Alice <a@example.com>", "Bob <b@example.com>"]
 ```
 
+If no other members are specified in [workspace.members filed](#the-members-and-exclude-fields) 
+in `Cargo.toml` in the workspace root directory, the root package will 
+become the default workspace member.
```

Comment:
For me, this feels a bit out of place with the section it is in.  Nothing else is talking about members or default members.  We also have a section down below for both members and default members which this pertains to.

Reply:
Since the main purpose here is to describe the possible default behavior of the root package, it might be more appropriate to put it here. 
You make a reasonable point, so I've added the description of the member in question earlier.